ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -667,11 +667,8 @@ EventVendor,string,Mandatory,Authentication,Enumerated,Salesforce|AWS|Microsoft|
 EventVendor,string,Mandatory,Common,,,
 EventVendor,string,Mandatory,Dhcp,,,
 EventVendor,string,Mandatory,Dns,Enumerated,Cisco|Corelight|GCP|Infoblox|Microsoft|Zscaler|Vectra AI|SentinelOne,
-EventVendor,string,Mandatory,FileEvent,Enumerated,Microsoft,
 EventVendor,string,Mandatory,NetworkSession,Enumerated,Fortinet|AppGate|Palo Alto|Microsoft|Zscaler|AWS|Vectra AI|WatchGuard|Cisco|Corelight|Check Point|Forcepoint|Cynerio|SentinelOne,
-EventVendor,string,Mandatory,Dns,Enumerated,Cisco|Corelight|GCP|Infoblox|Microsoft|Zscaler|Vectra AI,
 EventVendor,string,Mandatory,FileEvent,Enumerated,Microsoft|SentinelOne,
-EventVendor,string,Mandatory,NetworkSession,Enumerated,Fortinet|AppGate|Palo Alto|Microsoft|Zscaler|AWS|Vectra AI|WatchGuard|Cisco|Corelight|Check Point|Forcepoint|Cynerio,
```

Comment:
@azurekid can you please pull latest. Seems we don't have duplicate for NetworkSession

Reply:
Will check. This PR is already a bit older. Maybe someone else already fixed it.

Checking king right now

Reply:
> Will check. This PR is already a bit older. Maybe someone else already fixed it.
> 
> Checking king right now

Hi @vakohl (Varun),

There are a lot of duplicates in the file that were committed to the master.
I have run my validation script and fixed the issues.

![image](https://github.com/Azure/Azure-Sentinel/assets/40334679/af19655b-8820-49ef-b8c4-84615cccce04)

Reply:
Hey folks,

Another 12 days have past since the last request for change
Anything I can do to help decrease the workload?

Reply:
@azurekid can you please resolve the merge conflicts?

Reply:
Done